cli/tests/environment-list.bats

```diff
@@ -122,3 +122,17 @@ EOF
   assert_success
   assert_output "hello"
 }
+
```

Comment:
Did you mean to drop the integration test?

Reply:
I added this test as a regression test, but we can drop it if that's preferable/we assume the unit tet to cover this

Reply:
I think the unit test is probably sufficient